### DISCUSSION TITLE
Help with changing buttons colors

### DISCUSSION THREAD

**Discussion by nikster89** [rejetto/hfs#513] [Q&A]
Hello,

Can someone provide a code for custom html for the following elements please?

![interface colors](https://github.com/rejetto/hfs/assets/99842836/5011a56f-7595-45f0-a962-587c9bfc5cfb)

(tried the "inspect the element" option but most likely my codes was wrong)

**Comment by rejetto** [comment]
try this
```
<style>
body {
    --button-bg: red !important;
}
a {
    color: red !important;
}
</style>

```

**Reply by nikster89** [reply]
It works:)

Sorry to bother you again (and please remind your Paypal - I owe you for the help already !) but how to set colors for each element of menu?
![interface colors2](https://github.com/rejetto/hfs/assets/99842836/f00ac052-4b55-49bd-a38d-299482a2ba16)

Thank you in advance!!

**Reply by rejetto** [reply]
i tested the code i gave you, and it works also on those elements
<img width="400" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/b2f3be04-f3f6-4d87-8ade-449cb2f1c96c">

check if you have some code of yours preventing it.
don't worry about donations. My paypal is a@rejetto.com

**Comment by PH5HP** [comment]
Hi,

Here is what I did with some elelements on https://fileserver.ph5hp.nl
------------------------------------------------------------------------------
``

/* Navigation where are you    
.breadcrumb {
      background-color: #0000A0;
    }
    
/* Paging button background color */    
#paging button {
      background: #0000A0;
    }

/* link color */    
:root .theme-dark a {
      color: #ffffff;
}

body, input {
    color: #ffffff;
}

/* button color */
button {
    background-color: #0000A0;
    color: #ffffff;
}

/* alternate row background visibility */
ul.dir li:nth-of-type(odd) {
    background-color: #8888881a;
}

``
--------------------------------------------------------------------------------------

Use F12 in Google Chrome to see how each CSS element is called. If you how it works than it is easy.

Kind regards,
Henk - https://fileserver.ph5hp.nl

**Reply by nikster89** [reply]
Thank you!

the following code

<style>
body {
    --button-bg: #ed508c !important;
}
a {color: #525252
}

.breadcrumb {
background-color: #525252;
}
</style> 

worked,

maybe I will change the paging button later (via .pagingbutton parameter?)

Thank you again!
